------------------------------------------------------------------------
r18721 | buliabyak | 2008-05-24 21:32:40 +0200 (Sat, 24 May 2008) | 1 line

this hack does not seem to be needed with modern gtk, and it prevents remembering of last-browsed directory from working

Index: src/ui/dialog/filedialogimpl-gtkmm.cpp
===================================================================
--- src/ui/dialog/filedialogimpl-gtkmm.cpp	(revision 18720)
+++ src/ui/dialog/filedialogimpl-gtkmm.cpp	(revision 18721)
@@ -761,10 +761,6 @@
 bool
 FileOpenDialogImplGtk::show()
 {
-    Glib::ustring s = Glib::filename_to_utf8 (get_current_folder());
-    if (s.length() == 0)
-        s = getcwd (NULL, 0);
-    set_current_folder(Glib::filename_from_utf8(s)); //hack to force initial dir listing
     set_modal (TRUE);                      //Window
     sp_transientize((GtkWidget *)gobj());  //Make transient
     gint b = run();                        //Dialog
